bot/utils/functions.py

```diff
@@ -105,3 +109,23 @@ def format_user_list(users: List[UserProfile], title: str) -> str:
     response = f"📃 {title}:\n"
     response += "```\n" + "\n\n".join(user_lines) + "\n```"
     return response
+
+# async def check_clip_duration_and_permissions(segment: dict, message: Message) -> Optional[tuple]:
+#     start_time = max(0, segment["start"] - settings.EXTEND_BEFORE)
+#     end_time = segment["end"] + settings.EXTEND_AFTER
+#
+#     clip_duration = end_time - start_time
+#     if not await DatabaseManager.is_admin_or_moderator(message.from_user.id) and clip_duration > settings.MAX_CLIP_DURATION:
+#         await message.answer(get_limit_exceeded_clip_duration_message())
+#         return None
+#
+#     return start_time, end_time
```

Comment:
👀

Reply:
kurw miałem wyjebać to tylko tak na sekundę se zakomentowałem żeby zobaczyć gdzie mi wysra błędy XD

Reply:
jak coś ten cały komentarz jest do wyjebania ten kod nie zostaje